src/main/java/seedu/address/model/tutorial/TutorialName.java

```diff
@@ -28,7 +28,7 @@ public class TutorialName {
     public TutorialName(String name) {
         requireNonNull(name);
         checkArgument(isValidName(name), MESSAGE_CONSTRAINTS);
-        fullName = name;
+        fullName = name.toUpperCase();
```

Comment:
toUpperCase() in already handled in `src/main/java/seedu/address/logic/parser/tutorial/TutorialParserUtil.java`
More info: #224

Reply:
same as module code